src/test/java/unit/AcronymTransformerTest.java

```diff
@@ -36,15 +36,29 @@
                 arguments("chleb Itd.", "chleb I tak dalej"),
                 arguments("chleb iTd.", "chleb i Tak dalej"),
                 arguments("chleb itD.", "chleb i tak Dalej"),
-                arguments("chleb ITD.", "chleb I Tak Dalej")
+                arguments("chleb ITD.", "chleb I Tak Dalej"),
+                arguments("Druga była Dr Jones.", "Druga była Doktor Jones.")
         );
     }
 
     @ParameterizedTest(name = "should transform {0} into {1}")
     @MethodSource("transformsArgumentsProvider")
     void shouldTransform(String in, String out) {
         String transformed_text = new AcronymTransformer(new Text(in)).getText();
-
         Assertions.assertEquals(out, transformed_text);
     }
+
+    @Test
+    void acronymTransformerAcceptanceCriterion(){
+        Assertions.assertEquals("profesor", new AcronymTransformer(new Text("prof.")).getText());
+        Assertions.assertEquals("doktor", new AcronymTransformer(new Text("dr")).getText());
+        Assertions.assertEquals("na przykład", new AcronymTransformer(new Text("np.")).getText());
+        Assertions.assertEquals("i tym podobne", new AcronymTransformer(new Text("itp.")).getText());

```

Comment:
Jak ta linia się wywali to nie będziesz miał info który przypadek nie działa.
Gdyby dodać te przypadki poprzez test parametryzowany, widać który przypadek nie przechodzi. (W Intelij Idea tak widać, nie wiem jak to wygląda podczas uruchamiania testów z konsoli)

Reply:
Jest info, która pierwsza asercja nie przeszła w wygenerowanym raporcie w \target\surefire-reports